src/Estina/SmartFocus/Api/Rest/Notification.php

```diff
@@ -50,4 +56,116 @@ public function send(
 
         return $response;
     }
+
+    /**
+     * Send batch of email transactions at once.
+     *
+     * Also supports sending of dynamic content.
+     *
+     * @param SimpleXMLElement $xmlObject The xml object to post.
+     *
+     * @return string|false XML response or FALSE on failure
+     */
+    public function post(SimpleXMLElement $xmlObject)
```

Comment:
Can you rename this please to "sendObjects" to match SmartFocus docs: https://help-developer.smartfocus.com/#NMP_TRIGGER/sendObjects.htm.